standalone/examples.go

```diff
@@ -0,0 +1,46 @@
+package standalone
+
+import (
+	"encoding/json"
+	"io/ioutil"
+)
+
+// Example model of an example gRPC request
+type Example struct {
+	Name    string `json:"name"`
+	Request struct {
+		Timeout  string `json:"timeout"`
+		Metadata []struct {
+			Name  string `json:"name"`
+			Value string `json:"value"`
+		} `json:"metadata"`
+		Data map[string]interface{} `json:"data"`
+	} `json:"request"`
+	Service string `json:"service"`
+	Method  string `json:"method"`
+}
+
+// Examples list of gRPC request examples
```

Comment:
I don't think we need any of the stuff below this point in the file. This type and the helpers below needlessly broaden the surface area of the API without providing much value.